src/main/java/org/thymeleaf/cache/StandardCache.java

```diff
@@ -336,7 +336,7 @@ private void outputReportIfNeeded() {
                     long putCount = getPutCount();
                     long getCount = getGetCount();
 
-                    double hitRatio = hitCount / getCount;
+                    double hitRatio = (double) hitCount / (double) getCount;
```

Comment:
@yoshikawaa @danielfernandez Why not use `getHitRatio()` method to reduce duplication? This also adds a protection against division by zero.